bbc/simorgh
# Add Amharic translations for 500 and 404 pages

Resolves #3287 

**Overall change:** Adds Amharic translations for 500 and 404 pages.

**Code changes:**
- Added 500 and 404 translations to src/app/lib/config/services/amharic.js

**Testing notes:**

- To Observe the `404` page 
go to [http://localhost.bbc.com:7080/amharic/bbc_amharic_radio/blah](url)

To observe the 500 error page
- add `status = 500` here https://github.com/bbc/simorgh/blob/404Handling/src/app/routes/getInitialData/utils/fetchData/index.js#L16
- go to http://localhost.bbc.com:7080/amharic

Translations verified here: https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0

---
